cargo-bpf/src/build.rs

```diff
@@ -148,10 +155,20 @@ fn build_probe(cargo: &Path, package: &Path, target_dir: &Path, probe: &str) ->
 }
 
 pub fn build(
+    cargo: &Path,
+    package: &Path,
+    target_dir: &Path,
+    probes: Vec<String>,
+) -> Result<(), Error> {
+    build_ext(cargo, package, target_dir, probes, None)
+}
+
+pub fn build_ext(
     cargo: &Path,
     package: &Path,
     target_dir: &Path,
     mut probes: Vec<String>,
+    kernel_source_dir: Option<&Path>,
```

Comment:
I'd prefer getting rid of propagating `Option<>` parameters in functions, and instead wrapping default behaviour with specific implementations at the top of the call tree.